JavaTraining/BillsBurgers/src/Main.java

```diff
@@ -0,0 +1,63 @@
+public class Main {
+
+    public static void main(String[] args) {
+
+        Burger burger = new Burger("Simple Burger", "white roll", 5);
+        System.out.println(burger.getName() + " in " + burger.getBreadType());
+        burger.setCheese(true);
+        burger.setLettuce(true);
+        burger.setTomato(true);
+//        burger.receipt();
+//        System.out.println();
+        //burger.cheesePrice = 0;
+        burger.receipt();
+        System.out.println();
+
+
+        System.out.println();
+        HealthyBurger healthyBurger = new HealthyBurger("Healthy Burger", 6);
+        System.out.println(healthyBurger.getName() + " in " + healthyBurger.getBreadType());
+        healthyBurger.setTomato(true);
+        healthyBurger.setPeppers(true);
+        healthyBurger.receipt();
+
+        System.out.println();
+
+        DeluxeBurger deluxeBurger = new DeluxeBurger("Deluxe Burger", "seed roll", 5.5);
+        System.out.println(deluxeBurger.getName() + " in " + deluxeBurger.getBreadType());
+        deluxeBurger.receipt();
+        //deluxeBurger.lettuce = true; //QUE PASA SI HAGO ESTO?
```

Comment:
Technically, it'd still work because the field is protected and you're on the default package. However, it's all about encapsulation (https://www.thoughtco.com/definition-of-encapsulation-958068). The reason why we write getter and setter methods and some times make fields or certain methods private is such that we don't create a dependency on that piece of code. For example, let's say that today `lettuce` being true or false is all the software does, but if tomorrow we need some validation rules, you'd be bypassing those by setting the value directly. With a `setLettuce` method, you could run validations or even store the data in a different class or field altogether. It's all about evolution and extensibility without being tied to specific internals of a class. Hope it makes sense.

Reply:
A couple of additional links: [information hiding](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)#An_information-hiding_mechanism) and [OOP encapsulation]https://stackify.com/oop-concept-for-beginners-what-is-encapsulation/)

Reply:
Also comments in english... Its not just cultural imperialism (any more..  :P ) profesional software teams write the code in english (including comments) even if the team all share a non english language... Just a good habbit to get into

Reply:
yes, absolutely! the ugly comments I wrote are mostly because I'm studying and I'm adding notes and reminders and clarifications as I go, but I agree that if I do it now it could become a bad habit!